FilterViewController.m

```diff
@@ -27,11 +27,11 @@ - (void)viewDidLoad {
     MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
     
     //sends double array for filternames and single array for header lines
-    [labels setDataWithFilterNames:@[self.returnLabelArray] headerTitles:@[@"Labels"]];
+    [labels setDataWithFilterNames:self.returnLabelArray headerTitles:@[@"Labels"]];
```

Comment:
You didn't lead me astray, the manager can take a single array or a double array with one array as the first/only entry in the array for non hierarchy. Both these lines would work. The update that broke it was I made too many arrays out of Surveys. Array of double arrays didn't make the manager happy!

For this change I changed labels to be a single array since the code also takes that for a non hiearchy, just wanted to test that piece.